src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -133,7 +133,7 @@ export class CourseManagementRequests {
         releaseDate = day(),
         dueDate = day().add(1, 'day'),
         title = 'Programming ' + generateUUID(),
-        programmingShortName = 'cypress' + generateUUID(),
+        programmingShortName = 'programming' + generateUUID(),
```

Comment:
Just out of curiosity, why did you change from the cypress prefix to the programming and other prefixes, etc. ? I just know that in the pre-live Jira it might be helpful to see what's coming from cypress?!

Reply:
We want to stay consistent across the E2E test. Since they are deleted anyway, this is not an issue